src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise = new Promise((resolve, reject) => {
+  resolve(myDisplayer);
+
+  reject(myDisplayer2);
+});
```

Comment:
The promise is incorrectly implemented. The `resolve` and `reject` functions should be called with values, not functions. Additionally, calling both `resolve` and `reject` immediately is not a typical use case for promises. Consider revising this logic to simulate a real asynchronous operation.